{
  "totalNumberSkippingTests": 6,
  "totalMemory": 245366784,
  "bugType": "only_fail",
  "workspace": "./workspace",
  "totalNumberErroringTests": 0,
  "repo": "clun/ff4j",
  "freeMemory": 146001776,
  "bugCommit": "https://github.com/clun/ff4j/compare/c08dded9b876...4484821ee43d",
  "failingModule": "/root/workspace/clun/ff4j/226617056/ff4j-store-cassandra",
  "buildid": 226617056,
  "hostname": "repairnator",
  "totalNumberFailingTests": 4,
  "error-types": [
    "java.lang.AssertionError"
  ],
  "totalNumberRunningTests": 2306,
  "failing-test-cases": [
    {
      "className": "org.ff4j.cassandra.CassandraTest",
      "failingMethods": [
        "org.ff4j.cassandra.CassandraTest#org.ff4j.cassandra.CassandraTest"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Cassandra daemon did not start within timeout",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    },
    {
      "className": "org.ff4j.cassandra.EventRepositoryCassandraTest",
      "failingMethods": [
        "org.ff4j.cassandra.EventRepositoryCassandraTest#org.ff4j.cassandra.EventRepositoryCassandraTest"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Cassandra daemon did not start within timeout",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    },
    {
      "className": "org.ff4j.cassandra.PropertyStoreCassandraTest",
      "failingMethods": [
        "org.ff4j.cassandra.PropertyStoreCassandraTest#org.ff4j.cassandra.PropertyStoreCassandraTest"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Cassandra daemon did not start within timeout",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    },
    {
      "className": "org.ff4j.cassandra.FeatureStoreCassandraTest",
      "failingMethods": [
        "org.ff4j.cassandra.FeatureStoreCassandraTest#org.ff4j.cassandra.FeatureStoreCassandraTest"
      ],
      "erroringMethods": [],
      "failures": [
        {
          "failureName": "java.lang.AssertionError",
          "failureDetail": "Cassandra daemon did not start within timeout",
          "isError": false
        }
      ],
      "nbFailures": 1,
      "nbErrors": 0
    }
  ],
  "nbCPU": 4,
  "bugRepo": "clun/ff4j"
}